<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>abalone_gui</class>
 <widget class="QMainWindow" name="abalone_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>abalone_gui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="apply_move">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>800</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="move_alone">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>820</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>move_alone</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="move_group">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>820</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>move_group</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1171</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>750</y>
      <width>1251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Hello Welcome to abalone Game click on a choice below to defined the move you want to apply</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1252</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
